docs/about/migration/23-01.ipynb

```diff
@@ -0,0 +1,193 @@
+    "We give two examples:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ee57dea-da15-439b-8cde-17924a8f3289",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "\n",
+    "var = sc.ones(dims=['x', 'y'], shape=(4, 3))\n",
+    "norm = sc.array(\n",
+    "    dims=['y'],\n",
+    "    values=[0.1, 0.2, 0.3],\n",
+    "    variances=[0.1, 0.2, 0.3],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a3501a6-e855-4667-b2fc-a10a5a8bb669",
+   "metadata": {},
+   "source": [
+    "An operation between `var` and `norm` will raise `VariancesError` since `norm` is broadcast into dimension `'x'`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6f16545-caa4-453d-9e67-18663ca5281e",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "var / norm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0bd3114-3502-424f-a70b-ee7306264fae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.data.table_xyz(100).bin(y=3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65722b75-6aba-4a30-a9e3-2cbe1eb23394",
+   "metadata": {},
+   "source": [
+    "An operation between `binned` and `norm` will raise `VariancesError` since `norm` is broadcast to all bin elements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c33fce46-a1be-41aa-9660-c69b83a70548",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "binned / norm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9dd8781c-6003-4243-a1e3-810a4088219d",
+   "metadata": {},
+   "source": [
+    "## Working around the new behavior\n",
+    "\n",
+    "Scipp will not be able to support an automatic approach for handling these cases correctly, since this would be either very costly or very problem-specific.\n",
+    "An easy **but wrong** work around (equivalent to the old behavior in almost all cases) is to set the variances of the term being broadcast to `None`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "212a76f9-18b0-4390-a557-3c9cdd251c69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "stream_handler = logging.StreamHandler()\n",
+    "logger = sc.get_logger()\n",
+    "logger.addHandler(stream_handler)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "656d28de-d695-4523-a1c4-269853aa94ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger.warning(\n",
+    "    \"\"\"ATTENTION:\n",
+    "    Statistical uncertainties of the normalization term were IGNORED.\n",
+    "    This is probably incorrect.\"\"\"\n",
+    ")\n",
+    "norm.variances = None\n",
+    "var / norm\n",
+    "binned / norm"
```

Comment:
Why even show this code? We shouldn't make it too easy to use this approach.

Reply:
I don't see an alternative, since it might be months before people can implement another solution?

Reply:
So should they use wrong code in the interim period?

Reply:
Unless you have a better suggestion? Ideally you would determine a criterion for checking that the variances are small enough to justify this, but this will have to be done on a case-by-case basis.

Reply:
I would suggest to simply not give people a recipe for doing bad things. We can mention this approach but require them to think about the implementation at least a little.

Reply:
I am not sure I understand what you are suggesting?

Reply:
To remove the code in this section and simply say, 'If you need a quick and dirty, temporary solution, remove the variances. But this is wrong.' Only in a nicer way as you already did.

Reply:
Then people will do this, but not issue the warning, which is even worse? By giving an example they could at least copy this, so they will hopefully be constantly reminded of this when running the code.

Reply:
I doubt that this is how it will work out. But this discussion is pretty pointless. Leave it as it is.